fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanScheduleModelDisbursementPeriod.java

```diff
@@ -138,4 +138,14 @@ public void setEMIFixedSpecificToInstallmentTrue() {
     public boolean isEMIFixedSpecificToInstallment() {
         return isEMIFixedSpecificToInstallment;
     }
+
+    @Override
+    public BigDecimal rescheduleInterestPortion() {
+        return null;
```

Comment:
This looks curious.. is this intentional? Why?

Reply:
There is one of the two implimentations of the interface "LoanScheduleModelPeriod" (Disbursement period and Repayment Period).  These methods are not significant for Disbursement Period and needed only in Repayment Period, hence they are marked dummy.

Reply:
@vorburger : We can move away from this abstraction and change all the usages in a subsequent PR if you feel it's appropriate. Merging it for now.